### PR TITLE
Add IP-Adapter to docs

### PR DESCRIPTION
**Pull request by Millu**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [X] Documentation Update
- [ ] Community Node Submission

## Have you discussed this change with the InvokeAI team?
- [ ] Yes
- [ ] No, because:

      
## Have you updated all relevant documentation?
- [ ] Yes
- [ ] No

## Related Tickets & Documents

- Related Issue #
- Closes #

## Added/updated tests?

- [ ] Yes
- [ ] No : _please replace this line with details on why tests
      have not been included_

### CODE REVIEW

**Review by maryhipp — COMMENTED** [review]
It would be good to add a tooltip for this, too

File: docs/features/CONTROLNET.md [hunk 1]
```diff
@@ -132,3 +132,30 @@ Weight - Strength of the Controlnet model applied to the generation for the sect
...
+
+![IP-Adapter + T2I](https://github.com/tencent-ailab/IP-Adapter/raw/main/assets/demo/ip_adpter_plus_multi.jpg)
+![IP-Adapter + IMG2IMG](https://github.com/tencent-ailab/IP-Adapter/blob/main/assets/demo/image-to-image.jpg)
+
+#### Installation
+There are several ways to install IP-Adapter models with an existing InvokeAI installation:
+
+1. Through the command line interface launched from the invoke.sh / invoke.bat scripts, option [5] to download models
+2. Through the Model Manager UI with models from the *Tools* section of [www.models.invoke.ai](www.models.invoke.ai). To do this, copy the repo ID from the desired model page, and paste it in the Add Model field of the model manager. 
+3. Manually downloading the models files and placed in the `models/ip-adapter` folder of the Invoke root directory. **Note:** The image_encoder folder is necessary for IP-Adapter to function.
```
Comment (hipsterusername) [inline docs/features/CONTROLNET.md:149]: This is incorrect.

File: docs/features/CONTROLNET.md [hunk 2]
```diff
@@ -132,3 +132,30 @@ Weight - Strength of the Controlnet model applied to the generation for the sect
...
+[IP-Adapter](https://ip-adapter.github.io) is a tooling that allows for image prompt capabilities with text-to-image diffusion models. IP-Adapter works by analyzing the given image prompt to extract features, then passing those features to the UNet along with any other conditioning provided. 
+
+![IP-Adapter + T2I](https://github.com/tencent-ailab/IP-Adapter/raw/main/assets/demo/ip_adpter_plus_multi.jpg)
+![IP-Adapter + IMG2IMG](https://github.com/tencent-ailab/IP-Adapter/blob/main/assets/demo/image-to-image.jpg)
+
+#### Installation
+There are several ways to install IP-Adapter models with an existing InvokeAI installation:
+
+1. Through the command line interface launched from the invoke.sh / invoke.bat scripts, option [5] to download models
+2. Through the Model Manager UI with models from the *Tools* section of [www.models.invoke.ai](www.models.invoke.ai). To do this, copy the repo ID from the desired model page, and paste it in the Add Model field of the model manager.
```
Comment (hipsterusername) [inline docs/features/CONTROLNET.md:148]: This needs to be updated to include that the user needs to download the matching Encoder from the site as well.

**Review by hipsterusername — CHANGES_REQUESTED** [review]

File: docs/features/CONTROLNET.md [hunk 3]
```diff
@@ -96,6 +94,8 @@ A model that generates normal maps from input images, allowing for more realisti
 **Image Segmentation**: 
 A model that divides input images into segments or regions, each of which corresponds to a different object or part of the image. (More details coming soon)
 
+**QR Code Monster**:
```
Comment (hipsterusername) [inline docs/features/CONTROLNET.md:97]: Has this officially been added to the Installer?

File: docs/features/CONTROLNET.md (hunk 1 above)
Comment (Millu) [reply docs/features/CONTROLNET.md]: Updated

File: docs/features/CONTROLNET.md (hunk 3 above)
Comment (Millu) [reply docs/features/CONTROLNET.md:97]: Will be adding in a separate PR

**Review by hipsterusername — APPROVED** [review]

**Pull request merged**